hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Alert.java

```diff
@@ -230,13 +230,8 @@ public Long getLastResolvedTime() {
 
     @Override
     public String toString() {
-        return "Alert{" +
-                "severity=" + severity +
-                ", status=" + status +
-                ", notes=" + notes +
-                ", lifecycle=" + lifecycle +
-                ", resolvedEvalSets=" + resolvedEvalSets +
-                '}';
+        return "Alert [triggerId=" + getTriggerId() + ", severity=" + severity + ", status=" + status + ", tenantId="
```

Comment:
It was on purpose but it could be revised.  I was actually debugging and was annoyed that the triggerId for the alert was not being shown.  TenantId is important, severity and status were more fields I thought should be shown.  Lifecycle is a little verbose and resolvedEvalSets can be quite verbose.  I can add them back, but perhaps we should actually use log level to decide about the verbosity.  If log.isDebugEnabled { add on those two }.  What do you think?